### DISCUSSION TITLE
Are IDS Usage Control patterns still relevant?

### DISCUSSION THREAD

**Discussion by mabuse68** [eclipse-edc/.github#33] [Q&A] [answered]
Dear Community, here's the noob question of the day 😉

I am confused on the actual importance/support of patterns for Usage Control Policies.
Would it still be a safe bet to implement the patterns defined by IDS ([here](https://international-data-spaces-association.github.io/DataspaceConnector/Documentation/v6/UsageControl)), or should we expect [#878](https://github.com/eclipse-edc/Connector/discussions/878) to produce some concrete pattern implementations?
If this question has been answered somewhere else, I will appreciate if you can head me there please.

**Comment by jimmarino** [comment, marked as answer]
The goal of the EDC is to make usage control policies implementable - e.g., by providing the extensible policy engine - but not necessarily implement them. That's because operating environments may vary greatly. To take an example, "delete data" can mean many things such as deleting tables in a database, buckets in S3, etc. Many of the policies outlined in the referenced document are likely not able to be implemented in an automated way.

Given this, the goal of the EDC is to provide a framework for others to customize and adapt to specific use cases.
